packaging/linux/scripts/after-install-and-upgrade.sh

```diff
@@ -58,9 +58,14 @@ BK_IS_UBUNTU_14_10=$?
 command -v initctl > /dev/null
 BK_UPSTART_EXISTS=$?
 
-# Check if the system is Amazon linux which has an old and broken upstart.
-[ -f /etc/system-release ] && grep -qi "Amazon Linux" /etc/system-release
-BK_IS_AMAZON_LINUX=$?
+# Check if upstart is version 0.6.5 as seen on Amazon linux, RHEL6 & CentOS-6
+BK_UPSTART_TOO_OLD=0
+if [ $BK_UPSTART_EXISTS -eq 0 ]; then
+  BK_UPSTART_VERSION="$(initctl --version | awk 'BEGIN{FS="[ ()]"} NR==1{print $4}')"
+  if [ "$BK_UPSTART_VERSION" = "0.6.5" ]; then
```

Comment:
There is no version 0.6.4 :-)
http://upstart.ubuntu.com/cookbook/#releases
I was originally going to do a version check against the required minimum version for you upstart config. Alas, I don't know what version /is/ required. CentOS-6 and Amazon Linux both use exactly 0.6.5.
It may be that we can just look at the major version number but I don't have any machines on upstart 1.x